hieradata/development.yaml

```diff
@@ -196,6 +196,8 @@ govuk_containers::app::config::global_envvars:
 
 govuk_elasticsearch::version: '2.4.6'
 govuk_elasticsearch::cors_enabled: true
+govuk_elasticsearch::repo_path:
+  - '/var/govuk/govuk-puppet/development-vm/replication/backups'
```

Comment:
Can we get the base path from somewhere else in puppet? Hard-coding it looks a bit fragile.

Reply:
This is in line with existing usage in this file so feel it is fine for now.